src/apps/common/writers.py

```diff
@@ -0,0 +1,64 @@
+def empty_bytearray(preallocate: int) -> bytearray:
+    b = bytearray(preallocate)
+    b[:] = bytes()
+    return b
+
+
+def write_uint8(w: bytearray, n: int) -> int:
+    assert n >= 0 and n <= 0xFF
```

Comment:
I'm not sure it's worth it, but we can simplify these expressions to `assert 0 <= n <= 0xFF`

Reply:
👍